keep/api/tasks/process_event_task.py

```diff
@@ -394,3 +394,7 @@ def process_event(
     finally:
         session.close()
     logger.info("Event processed", extra=extra_dict)
+
+
+async def process_event(*args, **kwargs):
```

Comment:
Not required anymore because process_event is now a sync function (see main branch)